db/query.sql

```diff
@@ -33,10 +34,15 @@ SELECT EXISTS(SELECT 1 FROM candidate_records WHERE status = 'new' AND (metadata
 
 
 -- name: GetCandidateRecord :one
-SELECT * FROM candidate_records WHERE status = 'new' AND id = $1 LIMIT 1;
+SELECT * FROM candidate_records WHERE id = $1 LIMIT 1;
```

Comment:
I had to remove the `status = 'new'` condition here otherwise the `SetCandidateRecord` router middleware for restoring rejected suggestions would fail with a 404. I could make this `... WHERE status IN ('new', 'rejected') AND ...` but not sure if that doesn't make it more confusing by filtering specific statuses in a method with a generic name.